core/storage/statistics/gae_models.py

```diff
@@ -356,6 +356,29 @@ def create(cls, exp_id, user_id, rating, old_rating):
             old_rating=old_rating).put()
 
 
+class HintViewEventLogEntryModel(base_models.BaseModel):
```

Comment:
Hmm. Let's not do this yet -- can we remove all the "hint view" stuff from this PR (i.e. the storage, domain, etc. layers) and just focus on the changes to the exploration structure? The reason I say this is because I don't think we completely understand the end-to-end pipeline for hint views yet and how they will be aggregated/obsoleted, and these decisions impact how we design the storage layer. So let's figure that out and then put the whole thing in -- right now you have a rather piecemeal pipeline and it's not clear whether this is going to extend to a fully workable system.